vue/sbc-common-components/src/util/constants.ts

```diff
@@ -0,0 +1,13 @@
+export enum SessionStorageKeys {
+  KeyCloakToken = 'KEYCLOAK_TOKEN',
```

Comment:
BTW, Keycloak is 1 word (not cap C). I know, I know, it's abbreviated as KC 🤷‍♂

Reply:
Thanks.  Leaving as is for now.